examples/abrupt/KeYAbruptTerminationChallengeSilicon.java

```diff
@@ -0,0 +1,40 @@
+// -*- tab-width:4 ; indent-tabs-mode:nil -*-
+//:: cases KeYAbruptTerminationChallengeSilicon
+//:: tools silicon
+//:: verdict Fail
```

Comment:
I assume this is expected?

Reply:
Yes, it is a shortcoming of the silicon backend. Carbon verifies it just fine